### PR TITLE
[CF-110] Fix key error exception

### PR DESCRIPTION
**Pull request by SVilgelm**
Do not sort network interfaces if no data in instance_info_caches

### CODE REVIEW

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
Please update commit and pull request title with something meaningful, "Fix key error exception" is way too broad.

**Comment by roman-verchikov** [comment]
Alos feels like more like hiding the problem rather than resolving it

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

File: cloudferrylib/os/compute/nova_compute.py [hunk 1]
```diff
@@ -930,8 +930,14 @@ def get_networks(self, instance):
         if self.config.migrate.keep_network_interfaces_order:
             keys = (self.instance_info_caches.
                     enumerate_addresses(instance.id))
-            interfaces = sorted(interfaces,
-                                key=lambda i: keys[i['mac_address']])
+            try:
+                interfaces = sorted(interfaces,
+                                    key=lambda i: keys[i['mac_address']])
+            except KeyError:
+                LOG.warning("instance_info_caches table does not containt "
```
Comment (roman-verchikov) [inline cloudferrylib/os/compute/nova_compute.py]: "contain" not "containt"

File: cloudferrylib/os/compute/instance_info_caches.py [hunk 2]
```diff
@@ -52,4 +57,6 @@ def enumerate_addresses(self, instance_id):
         :return: Dictionary with mac address: position
         """
         network_info = self.get_network_info(instance_id)
+        LOG.debug('Netwrok info for instance %s: %s', instance_id,
```
Comment (roman-verchikov) [inline cloudferrylib/os/compute/instance_info_caches.py]: "Network", not "netwrok"

File: cloudferrylib/os/compute/nova_compute.py (hunk 1 above)
Comment (SVilgelm) [inline cloudferrylib/os/compute/nova_compute.py]: done

File: cloudferrylib/os/compute/instance_info_caches.py (hunk 2 above)
Comment (SVilgelm) [inline cloudferrylib/os/compute/instance_info_caches.py]: done

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by SVilgelm** [comment]
retest this please

**Comment by MirantisWorkloadMobility** [comment]
PEP8 check success, Unittests success

**Comment by roman-verchikov** [comment]
can be merged

**Pull request merged**